stream-chat-android-docs/src/main/kotlin/io/getstream/chat/docs/kotlin/Channels.kt

```diff
@@ -529,4 +530,45 @@ class Channels(val client: ChatClient, val channelClient: ChannelClient) {
                 }
         }
     }
+
+    /**
+     * @see <a href="https://getstream.io/chat/docs/android/slow_mode/?language=kotlin">Throttling & Slow mode</a>
+     */
+    inner class ThrottlingAndSlowMode {
+        private fun disableMessageSendingUi() {}
+        private fun enableMessageSendingUi() {}
+
+        fun enableAndDisable() {
+            val channelClient = client.channel("messaging", "general")
+
+            // Enable slow mode and set cooldown to 1s
+            channelClient.enableSlowMode(cooldownTimeInSeconds = 1).enqueue { /* Result handling */ }
+
+            // Increase cooldown to 30s
+            channelClient.enableSlowMode(cooldownTimeInSeconds = 30).enqueue { /* Result handling */ }
+
+            // Disable slow mode
+            channelClient.disableSlowMode().enqueue { /* Result handling */ }
+        }
+
+        fun blockUi() {
+            val channelClient = client.channel("messaging", "general")
+
+            // Get the cooldown value
+            channelClient.query(QueryChannelRequest()).enqueue { result ->
+                if (result.isSuccess) {
+                    val channel = result.data()
+                    val cooldown = channel.cooldown
+
+                    val message = Message(text = "Hello")
+                    channelClient.sendMessage(message).enqueue {
+                        // After sending a message, block the UI temporarily
+                        // The disable/enable UI methods have to be implemented by you
+                        disableMessageSendingUi()
+                        Handler().postDelayed(::enableMessageSendingUi, cooldown.toLong())
+                    }
```

Comment:
We could easily use `await()` and `delay()` in Kotlin snippets - we'll have to make a decision about this at some point, for now we have `enqueue` and callback lambdas everywhere

Reply:
Maybe put handler as a class level property?
``` kotlin
private val uiHandler = Handler(Looper.getMainLooper())
```

Reply:
Things outside the method won't be copied to the docs, so I'd rather create it here, otherwise it'll be a mystery where that variable comes from.

For the threading part, since `enqueue` runs the callback on the UI thread, the Handler will be on the UI thread too.

Reply:
I understand why usage `Handler` in enqueue is not dangerous. It's just not obvious and can lead to inappropriate usage in `callback` methods for example

Reply:
Made the loopers explicit

Reply:
I see it simple enough
Adding `await()` and `dealay()` will forze you to use suspend functions that could be a bit confuse when browsing our documentation